Tasks/VsTest/vstest.ts

```diff
@@ -343,8 +350,249 @@ function getTestResultsDirectory(settingsFile: string, defaultResultsDirectory:
     return defer.promise;
 }
 
+function getTICollectorURI () {
+    return "datacollector://microsoft/TestImpact/1.0";
+}
+
+function getTIAssemblyQualifiedName(vsVersion) {
+    return "Microsoft.VisualStudio.TraceCollector.TestImpactDataCollector, Microsoft.VisualStudio.TraceCollector, Version=" + vsVersion + ".0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a";
+}
+
+function getTIFriendlyName() {
```

Comment:
Created function so that if we need to change the behaviour based on input, it can be easily done.